src/request/request.model.ts

```diff
@@ -22,17 +20,17 @@ export class Request extends Model<Request, CreateRequest> {
   id: number;
 
   @ForeignKey(() => User)
-  @Column({ type: DataType.NUMBER, allowNull: false })
+  @Column({ type: DataType.INTEGER, allowNull: false })
   from: number;
 
   @ForeignKey(() => User)
-  @Column({ type: DataType.NUMBER, allowNull: false })
+  @Column({ type: DataType.INTEGER, allowNull: false })
   to: number;
 
-  @Column({ type: DataType.NUMBER, defaultValue: RequestStatus.pending})
+  @Column({ type: DataType.INTEGER, defaultValue: RequestStatus.PENDING})
```

Comment:
Do you have foreign keys for that col? can it rather be enum val than just number?